release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2547,9 +2547,12 @@ Workaround: Use OpenShift OAuth for authentication. (link:https://bugzilla.redha
 and recreate it without a BFD profile. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2050824[*BZ#2050824*])
 
 * For clusters that run on {rh-openstack} and use Mellanox NICs as part of a single-root I/O virtualization configuration (SR-IOV), you may not be able to create a pod after you start one, restart the SR-IOV device plugin, and then stop the pod. No workaround is available for this issue.
-
 // TODO: ^^^ Add BZ when created.
 
+* {product-title} supports deploying an installer-provisioned cluster without a DHCP server. However, without a DHCP server, the bootstrap VM does not receive an external IP address for the `baremetal` network. (link: https://bugzilla.redhat.com/show_bug.cgi?id=2048600[*BZ#2048600*]) To assign an IP address to the bootstrap VM, see link:https://access.redhat.com/articles/6850661[Assigning a bootstrap VM an IP address on the baremetal network without a DHCP server].
+
+* {product-title} supports deploying an installer-provisioned cluster with static IP addresses on the `baremetal` network for environments without a DHCP server. If a DHCP server is present, nodes might retrieve an IP address from the DHCP server on reboot. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2036677[*BZ#2036677*]) To prevent DHCP from assigning an IP address to a node on reboot, see link:https://access.redhat.com/articles/6865841[Preventing DHCP from assigning an IP address on node reboot].
```

Comment:
I think we usually prefer the (\<BZ link>) to be at the very end of the paragraph (after the "see" links). Same for the above item.